CHANGELOG.adoc

```diff
@@ -67,6 +67,8 @@ The help text can include multiple lines (separated by `\n`), but no text format
 - https://github.com/eclipse-sirius/sirius-components/issues/2083[#2083] [diagram] Add support for edge markers
 - https://github.com/eclipse-sirius/sirius-components/issues/2086[#2086] [diagram] Add a panel with the default diagram actions
 - https://github.com/eclipse-sirius/sirius-components/issues/2087[#2087] [diagram] Add support for snap to grid
+- https://github.com/eclipse-sirius/sirius-components/issues/2115[#2115] [form] Add the possibility to choose the label position on checkbox widget.
```

Comment:
A screenshot showing the previous style/layout and the new one(s) side by side would be nice.

Reply:
I've added before/after screenshots